articles/creating-a-hangman-game-with-vanilla-javascript/index.md

```diff
@@ -0,0 +1,238 @@
+#Creating a Hangman Game with Vanilla JavaScript
+A Hangman game is a word-guessing game. One player picks a secret word, and the other player tries to guess it. In this article, I will take you through building a simple Hangman game with just Vanilla JavaScript, and more importantly, how I made it accessible for players who navigate with their keyboard without using any frameworks.
+Prerequisites
+Any good Text Editor.
+Basic knowledge of HTML, Bootstrap, and JavaScript.
+##Designing the Game
+I majorly used Bootstrap CDN in styling this game. Here is what the code looks like. The images used in this project can be found in my Github repo.
```

Comment:
Provide instructions. ie. Tell the reader to create a file, then do something after that. Don't make the reader think of what to do with the code. 

Please revise your document to fix this.

Also, provide the direct link to the images in your GitHub repo.